src/commands/prepare.ts

```diff
@@ -277,7 +281,7 @@ function checkGitStatus(repoStatus: StatusResult, rev: string) {
   ) {
     reportError(
       'Your repository is in a dirty state. ' +
-        'Please stash or commit the pending changes.',
+      'Please stash or commit the pending changes.',
```

Comment:
this looks like formatter noise? I would probably rebase this out and/or we should set up linters that make this consistent